src/ucx_van.h

```diff
@@ -800,12 +800,13 @@ class UCXVan : public Van {
   int Bind(Node &node, int max_retry) override {
     contexts_.reserve(node.num_ports);
 
-    UCX_LOG(2, "Start/Bind UCX Van, num ports " << node.num_ports);
+    UCX_LOG(1, "Start/Bind UCX Van, num ports " << node.num_ports);
 
     // Create separate UCX context for every device. If device is GPU, set the
     // corresponding cuda device before UCX context creation. This way UCX will
     // automatically select the most optimal NICs for using with this device.
     for (int i = 0; i < node.num_ports; ++i) {
+      node.dev_ids[i] = i;
```

Comment:
why need to set it in the UCX van? Also why do you set it to port index? Should not it be assigned to real device ids?

Reply:
@eric-haibin-lin 

this is a temporary workaround so I could run the test with multiple GPUs. the port index is used as GPU ids here, and the logical GPU ids are controlled by `CUDA_VISIBLE_DEVICES`.  Since the ucx van is the only van that supports GPUs, the logic to assign device ids and device types will be in the ucx van instead of van.cc. Also I just realized `node.dev_types[i]` is always 0 here.